libraries/AP_Arming/AP_Arming.cpp

```diff
@@ -1216,6 +1217,18 @@ bool AP_Arming::visodom_checks(bool display_failure) const
     return true;
 }
 
+// check disarm switch is asserted
+bool AP_Arming::disarm_switch_checks(bool display_failure) const
+{
+    const RC_Channel *chan = rc().find_channel_for_option(RC_Channel::AUX_FUNC::DISARM);
+    if (chan != nullptr && chan->get_aux_switch_pos() == RC_Channel::aux_switch_pos_t::HIGH){
```

Comment:
Should this be checking `!= RC_Channel::aux_switch_pos_t::LOW`?

Reply:
I think it's OK to be able to arm in the MIDDLE position.
https://github.com/ArduPilot/ardupilot/pull/14244#discussion_r418537445

Reply:
I think I'm going to disagree on this one, but that's not a blocker.  Other devs may want to chime in.

Leaving the middle switch position "reserved for future use" won't cost us anything at this point, and it makes for a cleaner story to restrict this to just the two positions (`HIGH`/`LOW`).